java/src/jmri/jmrix/lenz/swing/lv102/LV102InternalFrame.java

```diff
@@ -560,15 +560,17 @@ void writeRailComTimingSetting(Programmer opsProg) {
     }
 
     // Set to LV102 default values.  Voltage is 16, E Line is Active, 
-    // Railcom is innactive, Railcom Mode is 3 bit cutout.
+    // Railcom is inactive, Railcom Mode is 3 bit cutout, Railcom Timing is 435 uS.
     void defaultLV102Settings() {
         voltBox.setSelectedIndex(10);
         eLineBox.setSelectedIndex(0);
         railComBox.setSelectedIndex(1);
         railComModeBox.setSelectedIndex(0);
+        // also set RailCom Timing, 435uS is named Default
```

Comment:
That would be ok.

The early LZV100/LV102 didn't have setting the railcom timing as an option, which is why that is missing.